cloudbank-v4/account/src/main/resources/application.yaml

```diff
@@ -76,10 +87,21 @@ management:
   otlp:
     tracing:
       endpoint: ${otel.exporter.otlp.endpoint}
+  observations:
+    key-values:
+      app: ${spring.application.name}
 
 logging:
   level:
     root: INFO
-    com.example: INFO
+    com.example: DEBUG
```

Comment:
do we really need debug on com.example?  seems a bit high?

Reply:
there are not enough INFO logs corresponding to traces, trying to see how can optimize